.changeset/popular-roses-deliver.md

```diff
@@ -0,0 +1,5 @@
+---
+'@makeswift/runtime': patch
+---
+
+Add RichText control with naive selection
```

Comment:
I think being a tad more descriptive would be helpful. It's not clear to me what "naive selection" is